pro/src/pages/Home/Homepage.tsx

```diff
@@ -63,6 +63,9 @@ const Homepage = (): JSX.Element => {
             dateCreated: '',
             hasAvailablePricingPoints: false,
             hasDigitalVenueAtLeastOneOffer: false,
+            hasValidBankAccount: false,
+            hasPendingBankAccount: false,
+            venuesWithNonFreeOffersWithoutBankAccounts: [],
```

Comment:
Histoire de ne pas afficher de bannière en cas d'erreur.
```suggestion
            hasValidBankAccount: true,
            hasPendingBankAccount: false,
            venuesWithNonFreeOffersWithoutBankAccounts: [],
```